src/pages/quiz/Quiz.tsx

```diff
@@ -80,7 +91,19 @@ const QuizPage = () => {
     items.filter(({ type }) => type !== QuestionTypesEnum.OpenAnswer),
     data
   )
+  const getQuizzes = useCallback(() => {
+    if (quizId && id) {
+      return ResourceService.getFinishedQuizzesByQuizId(id, quizId)
+    }
+  }, [id, quizId])
 
+  const { data: finishedAttempts } = useQuery({
+    queryKey: ['quizzes', id, quizId],
+    queryFn: getQuizzes
+  })
```

Comment:
```suggestion
  })
  
```
Add an indent here, please.